docs/jaeger-support.asciidoc

```diff
@@ -53,6 +53,34 @@ Based on the endpoint enabled in the previous step, configure the relevant host
 * `apm-server.jaeger.grpc.host` defaults to `localhost:14250`.
 * `apm-server.jaeger.http.host` defaults to `localhost:14268`.
 
+[[jaeger-configure-sampling]]
+*Configure Sampling*
+
+The gRPC endpoint supports probabilistic sampling, which can be used to reduce the amount of data that your agents collect and send.
+Probabilistic sampling makes a random sampling decision based on the configured sampling value.
+For example, a value of `.2` means that 20% of traces will be sampled.
+
+By default, when using Jaeger Clients over gRPC, two things happen:
```

Comment:
Jaeger Agents talk to the APM Server via gRPC, Jaeger clients can only talk to the APM Server via thrift over http. Sampling is only supported for the gRPC endpoint.